bindings/uniffi/Cargo.toml

```diff
@@ -7,20 +7,14 @@ repository.workspace = true
 license-file.workspace = true
 
 [dependencies]
-credentials = { path = "../../crates/credentials" }
-crypto = { path = "../../crates/crypto" }
-dids = { path = "../../crates/dids" }
-jwk = { path = "../../crates/jwk" }
-jwt = { path = "../../crates/jwt" }
-keys = { path = "../../crates/keys" }
+web5 = { path = "../../crates/web5" }
 uniffi = { version = "0.27.1", features = ["cli"] }
 
 [build-dependencies]
 uniffi = { version = "0.27.1", features = ["build"] }
 
 [lib]
 crate-type = ["cdylib"]
-name = "web5"
```

Comment:
FYI this is relevant to the binding scripts https://github.com/TBD54566975/web5-rs/pull/227